bookmarks.html

```diff
@@ -8,58 +8,151 @@
     <link rel="stylesheet" href="./styles/index.css" />
   </head>
   <body class="app-grid">
-    <h1 class="header__title">Bookmarks</h1>
+    <h1 class="app-grid__header title" id="title">Quiz-App</h1>
+    <main class="app-grid__main">
+      <section class="card">
+        <article class="card__box">
+          <h2 class="card__title">Question:</h2>
+          <button class="card__bookmark">
+            <svg
+              class="card__icon card__icon--red"
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 24 24"
+              width="32"
+              height="32"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            >
+              <path d="M19 21l-7-5-7 5V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2z" />
+            </svg>
+          </button>
+          <p class="card__text card__text--blue">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Et libero
+            doloremque, laboriosam velit deleniti repudiandae.
+          </p>
+          <button class="card__answer">Show answer</button>
+          <ul class="tag__list tag__list--card">
+            <li class="tag__item tag__item--card">Tag1</li>
+            <li class="tag__item tag__item--card">Tag2</li>
+            <li class="tag__item tag__item--card">Tag3</li>
+            <li class="tag__item tag__item--card">Tag4</li>
+          </ul>
+        </article>
+      </section>
 
-    <main class="app-grid__scrollarea">
-      <article class="card">
-        <h2 class="card__title">Question</h2>
-        <!-- icon color to red (active) -->
-        <i class="card__bookmark">BOOKMARK ICON or IMAGE</i>
-        <p class="card__question">
-          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Et libero
-          doloremque, laboriosam velit deleniti repudiandae.
-        </p>
-        <!-- maybe a button is the better choice - depends on js or not
-        <button class="button">Show Answer</button> -->
-        <details class="card__button">
-          <summary class="card__show-answer">Show Answer</summary>
-          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Amet,
-          quibusdam. Tenetur pariatur repudiandae maiores minus officiis a optio
-          consequuntur nihil.
-        </details>
-        <div class="card__answers">
-          <ul class="card__list">
-            <a class="card__link" href="">
-              <li class="card__item">A</li>
-            </a>
-            <a class="card__link" href="">
-              <li class="card__item">B</li>
-            </a>
-            <a class="card__link" href="">
-              <li class="card__item">C</li>
-            </a>
-            <a class="card__link" href="">
-              <li class="card__item">D</li>
-            </a>
+      <section class="card">
+        <article class="card__box">
+          <h2 class="card__title">Question:</h2>
+          <button class="card__bookmark">
```

Comment:
Okay you switch from details-list to a button, me too: But now Im thinking about adding also a solution with a detail-list. Just dont know how to go on from here with JS next week- Think both will work ;)